### DISCUSSION TITLE
Did you see this from Hardware Haven?

### DISCUSSION THREAD

**Discussion by Dvalin21** [roflcoopter/viseron#731] [General]
@roflcoopter  https://youtu.be/6ahtCpIoWU4?si=-_o6rdK7zwnATpQY

**Comment by roflcoopter** [comment]
Oh cool! Thanks for the heads up.
A bit bummed his wide angle camera didnt work properly :(

**Reply by Dvalin21** [reply]
Maybe you can reach out and do a collaboration?

**Comment by Dvalin21** [comment]
True, but now you know its something you can work on. And if I could add,
some newer cameras have three lenses, able to get a stream from each one.
It would be awesome to be able to view all three and get detection from all
three.
